packages/video_player/video_player_platform_interface/lib/video_player_platform_interface.dart

```diff
@@ -49,7 +49,8 @@ abstract class VideoPlayerPlatform extends PlatformInterface {
   }
 
   /// Creates an instance of a video player and returns its textureId.
-  Future<int?> create(DataSource dataSource) {
+  Future<int?> create(
+      DataSource dataSource, VideoPlayerOptions? videoPlayerOptions) {
```

Comment:
good idea, do you have an idea about a suitable name for the new create method? It would then be something like: 'Future<int?> newCreate(
      DataSource dataSource, CreationParameters? creationParameters) {'

Reply:
I've made a createWithParameters method